docs/source/modules/damping_driver.rst

```diff
@@ -0,0 +1,139 @@
+damping_driver.f90
+======================
+
+Summary
+-------
+The ``damping driver`` module is called by the ``idealized_moist_phys`` module by setting ``do_damping`` to true. It controls the upper level momentum damping in Isca. It controls 4 functions:
+
+1. **Rayleigh friction** which acts at levels ``1 to kbot``. This function is located in ``damping_driver`` itself.
+2. A (orographic) **mountain gravity wave drag** module may be called. This is the ``mg_drag`` module.
+3. A (non orographic) **convective gravity wave drag** module may be called. This is the ``cg_drag`` module.
```

Comment:
Suggest simplification: A blabla module (``name``) may be called (optional).

Reply:
Yeah that's much better, sorted.